index.html

```diff
@@ -27,7 +27,7 @@
       ga('create', 'UA-63938505-1', 'auto');
       ga('send', 'pageview');
     </script>
-
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
```

Comment:
JQuery is a "*crutch*", I know... but it makes our code cleaner and easier for new people to read...